docs/docs/api-tour.mdx

```diff
@@ -170,23 +170,23 @@ Technically speaking, these are ed25519 keypairs encoded in base32.
 
 ## Query
 
-<img src="/api-tour/query.svg" class="bg-gray-200 p-4"/>
+<img src="/api-tour/query.svg" class="bg-gray-200 p-4" alt="A graphical representation of a query" />
 
 Used to describe a document query. Used with `ReplicaAsync.query` and `QueryFollower`.
 
 <DocLink apiName="Query"/>
 
 ## Doc
 
-<img src="/api-tour/doc.svg" class="bg-gray-200 p-4"/>
+<img src="/api-tour/doc.svg" class="bg-gray-200 p-4" alt="A grapichal representation of a doc" />
```

Comment:
Tiny typo.

Reply:
Oops. Thanks for spotting it! Should be resolved now.